src/main/java/net/neoforged/neoforge/common/extensions/IBlockExtension.java

```diff
@@ -230,6 +232,29 @@ default boolean onDestroyedByPlayer(BlockState state, Level level, BlockPos pos,
         }
     }
 
+    /**
+     * Called when a player removes a block. This is responsible for
+     * actually destroying the block, and the block is intact at time of call.
+     * This is called regardless of whether the player can harvest the block or
+     * not.
+     *
+     * Return true if the block is actually destroyed.
+     *
+     * This function is called on both the logical client and logical server.
+     *
+     * @param state       The current state.
```

Comment:
While you're at it, can you please reorder these parameter tags to match the order of the parameters declared by the method